projects/plugins/jetpack/_inc/lib/core-api/wpcom-fields/post-fields-publicize-connections.php

```diff
@@ -251,6 +251,9 @@ public function get( $post_array, $request ) { // phpcs:ignore VariableAnalysis.
 			$output_connection['id']            = (string) $connection['unique_id'];
 			$output_connection['connection_id'] = (string) $connection['id'];
 
+			$output_connection['can_disconnect'] = Publicize::can_manage_connection( $connection );
```

Comment:
Shall we change this to `current_user_can( 'edit_others_posts' ) || get_current_user_id() === (int) $connection_data['user_id'];` till the test initialization gets fixed?

Reply:
Will probably have to do the same thing in the package as well.

Reply:
Done. Thanks